pkgs/applications/graphics/inkscape/default.nix

```diff
@@ -106,9 +114,11 @@ stdenv.mkDerivation rec {
     python3Env
     zlib
   ] ++ stdenv.lib.optionals (!stdenv.isDarwin) [
-    gtkspell2
+    gdl
+    gtkspell3
   ] ++ stdenv.lib.optionals stdenv.isDarwin [
     cairo
+    gtk-mac-integration
```

Comment:
Blegh, they still did not switch from `GtkApplication` and `GtkosxApplication` fully to `GApplication`.

Reply:
:shrug: